services/QuillLMS/client/app/bundles/Staff/components/comprehension/activitySessions/sessionView.tsx

```diff
@@ -78,13 +102,13 @@ const SessionView = ({ match }) => {
       <Switch>
         <Redirect exact from='/activities/:activityId/activity-sessions/:sessionId' to='/activities/:activityId/activity-sessions/:sessionId/overview' />
         {/* eslint-disable-next-line react/jsx-no-bind */}
-        <Route component={() => <SessionOverview activity={activity} sessionData={sessionData} />} path='/activities/:activityId/activity-sessions/:sessionId/overview' />
+        <Route component={() => <SessionOverview activity={activity} rules={rules} sessionData={sessionData} />} path='/activities/:activityId/activity-sessions/:sessionId/overview' />
```

Comment:
I don't think we need to be passing a rules prop here?

Reply:
The overview page actually shows all three of the sub-items, but as far as I can tell it does need this data passed through.